portal/src/content/docs/documentation/working-with-tables.md

```diff
@@ -0,0 +1,101 @@
+
+![SORTING](../../../assets/table-sorting.png)
+
+The pagination at the bottom helps to navigate to different pages of the tabular data. You can also directly navigate to specific pages as well. Additionally, the refresh button allows you to reload the current page.
+
+![PAGINATION](../../../assets/table-pagination.png)
+
+The "Results per page" dropdown button allows you to specify the numbers of rows you want to be displayed.
+
+## Adding/Editing Metadata
+
+You can edit the metadata of the tabular resource by clicking on 'Metadata' menu in the toolbar. To hide the 'Metadata' window, simply click the menu again.
+
+![METADATA](../../../assets/table-metadata-menu.png)
+
+The metadata editor provides a user friendly graphical interface to modify metadata such as licenses, contributors, dialect, schema etc. It has three sections: a menu explorer, an input form and a help window.
+
+![METADATA WINDOW](../../../assets/table-metadata-window.png)
+
+The help window, on the right side of metadata editor, displays info about each input element when selected. The metadata editor generates a resource specification for the selected tabular resource. The metadata adheres to the [Frictionless Data Standards](https://specs.frictionlessdata.io). 
+
+## Finding Errors
+
+The editor applies [baseline checks](https://framework.frictionlessdata.io/docs/checks/baseline.html) to validate the data in the table resource. If any errors are detected, they will be displayed with a red background as shown in the figure below.
+
+![TABLE INVALID](../../../assets/table-invalid.png)
+
+To view detailed error, simply click the 'Report' menu. You will see the detailed description for each error at the bottom. To hide the 'Report' window, simply click the menu again.
+
+![TABLE REPORT](../../../assets/table-report.png)
+
+For table without errors, you will see a green background with a 'VALID' check mark .
+
+![TABLE VALID](../../../assets/table-valid.png)
+
+## Fixing Errors
+
+To fix cell errors, you can directly edit the data cells in the viewer/editor. When you alter a cell's value, it changes to yellow, indicating a modified cell.
+
+![INVALID FILE BEFORE](../../../assets/table-invalid-before.png)
+
+For schema changes, navigate to the 'Schema Editor > Fields' menu within the 'Metadata' menu.
+
+![INVALID FILE AFTER FIXING ERRORS](../../../assets/table-invalid-after.png)
+
+To make other changes like headers, blank rows, etc., you can make direct edits to your files.
+
+## Saving
+
+You can modify the data by clicking in the table's cell and editing the content. The changes can be saved using 'Save' button. When changes are made, the 'Save' and 'Revert' button gets activated at the bottom of the editor. If needed, you can revert the changes clicking the 'Revert' button.
+
+![TABLE EDIT](../../../assets/table-edit.png)
+
+## Adding Package Metadata
```

Comment:
Let's don't mention packaging here to simplify future maintenance. We can remove `adding package metadata` paragraph here and use `Publish` button directly - it will package table automatically